app/models/check.rb

```diff
@@ -7,7 +7,7 @@ class Check < ApplicationRecord
   belongs_to :link
 
   scope :created_within, ->(within) { where("created_at > ?", Time.zone.now - within) }
-  scope :requires_checking, -> { where(started_at: nil).or(Check.where(completed_at: nil).where("created_at < ?", RECHECK_THRESHOLD.ago)) }
+  scope :requires_checking, -> { where(started_at: nil).or(self.where(completed_at: nil).where("created_at < ?", RECHECK_THRESHOLD.ago)) }
```

Comment:
Rubocop suggests removing the `self`:
`scope :requires_checking, -> { where(started_at: nil).or(where(completed_at: nil).where("created_at < ?", RECHECK_THRESHOLD.ago)) }`